### ISSUE TITLE
Overlapping image codings

### ISSUE THREAD

**Issue by AndrzejWawa** [ccbogel/QualCoder#644]
When there are overlapping codings of image and you try to unmark one of them, create memo or add/remove important mark, there is no dialog box to select coding to unmark, memoing, or adding/removing mark. This box is present in the text coding mode and would be very helpful in the case of images.

Moreover, when writing memo, there should be information on name of code of coding for which the memo is. This is present in the text coding mode.

**Comment by ccbogel** [comment]
OK I have done most of this.
The not important secon added comments. if it becomes important, create a new issue.

**Comment by AndrzejWawa** [comment]
When right-hand clicking on the overlapping codes I have this error:

![obraz](https://user-images.githubusercontent.com/77200214/175820493-0c7d8b47-95e3-4b58-a1a5-e2a60c5f7d72.png)

**Comment by ccbogel** [comment]
Is this problem still occurring - I thought  I had fixed it.

**Comment by AndrzejWawa** [comment]
Yes, it is still the problem. Right-click selection of codings work properly in the case of previously added codings. However, when you add new coding and after this right-click on it, the following communication appears:

![obraz](https://user-images.githubusercontent.com/77200214/175905821-05051964-60e2-40cb-8d93-36c78781bab8.png)

The problem is solved after reopening the image codings mode.

**Comment by ccbogel** [comment]
now fixed. thx.
